doc/building-and-deploying-the-website.md

```diff
@@ -0,0 +1,22 @@
+## Building and Deploying (Automated)
+
+### Testing
+
+After you are finished developing and making your changes, make sure to test them. Run:
+
+	npm test
+
+`npm test` runs both [eslint] and [mocha] tests. If your tests pass, commit and push your work to Github.
+
+### Travis 
+
+[Travis] automatically builds and publishes the website on every change. Travis will build the full website for you by running `gulp build --prod` under the hood. Travis also uses [SVN] to update, copy, add, and commit the new changes over to the website. Committing to svn can only occur once the commit has been merged to master. 
```

Comment:
Maybe add some information how Travis is setup and configured (after the first sentence maybe?).